src/App.tsx

```diff
@@ -18,6 +18,9 @@ function App() {
   let nextHighTide = "tomorrow";
   let nextLowTide = "tomorrow";
 
+  let nextTideTime: "tomorrow" | Date = "tomorrow";
```

Comment:
This is the quick and dirty version, just to get data flowing through the components. Will need to clean up the logic and start adding a bit more architecture to optimize re-renders, cache the api data to be a good citizen and prevent rate limits, etc.